dev-docs/bidders/tagoras.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: bidder
+title: Tagoras
+description: Prebid Tagoras Bidder Adaptor
+biddercode: tagoras
+filename: tagorasBidAdapter
+userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrableId, pubCommonId, unifiedId
+tcfeu_supported: true
+usp_supported: true
+coppa_supported: false
+schain_supported: true
+gpp_supported: true
+floors_supported: true
+media_types: banner, video
+prebid_member: false
+safeframes_ok: false
+deals_supported: false
+pbs_app_supported: false
+fpd_supported: false
+ortb_blocking_supported: false
+multiformat_supported: will-bid-on-one
+pbjs: true
+sidebarType: 1
+---
+
+### Bid Params
+
+{: .table .table-bordered .table-striped }
+| Name       | Scope    | Description                                                                               | Example                      | Type     |
+|------------|----------|-------------------------------------------------------------------------------------------|------------------------------|----------|
+| `cId`      | required | The connection ID from Tagoras.                                                          | `'562524b21b1c1f08117fc7f9'` | `string` |
+| `pId`      | required | The publisher ID from Tagoras.                                                           | `'59ac17c192832d0011283fe3'` | `string` |
+| `bidFloor` | optional | The minimum bid value desired. Tagoras will not respond with bids lower than this value. | `0.90`                       | `float`  |
```

Comment:
Prebid strongly discourages bidder-specific floors because it's a headache for publishers, but since your adapter does the right thing and pays attention to the global floor, this is not a blocker.